test/augmentation/test_torchaudio.py

```diff
@@ -64,6 +76,24 @@ def test_volume_does_not_change_num_samples(audio):
         assert augmented_audio != audio
 
 
+def test_reverb_does_not_change_num_samples(audio, rir):
+    augment_fn = ReverbWithImpulseResponse(rir=rir)
+    for _ in range(10):
+        augmented_audio = augment_fn(audio, sampling_rate=SAMPLING_RATE)
+        assert augmented_audio.shape == (1, 16000)
+
+
+@pytest.mark.parametrize("normalize_output", [True, False])
+def test_reverb_normalize_output(audio, rir, normalize_output):
+    augment_fn = ReverbWithImpulseResponse(rir=rir, normalize_output=normalize_output)
+    orig_energy = np.sum(np.abs(audio) ** 2) / audio.shape[1]
+    for _ in range(10):
+        augmented_audio = augment_fn(audio, sampling_rate=SAMPLING_RATE)
+        rvb_energy = np.sum(np.abs(augmented_audio) ** 2) / augmented_audio.shape[1]
+        if normalize_output:
```

Comment:
should we assert something when `normalize_output` is `False`? Do you expect the reverb-ed signal energy to be higher in that case?

Reply:
Hmm, TBH I am not sure what to expect. I borrowed this flag from the Kaldi implementation, where it is set to True by default. I haven't seen any uses where it is called with this flag set to false, but I would think the energy would be higher in that case.